### PR TITLE
LG-1335: Update Dashboard to use Identity Design System

### PR DESCRIPTION
**Pull request by karlaturcios**
This is a work in progress of the login design style guide implementation on the dashboard app

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -1,131 +1,208 @@
 <!doctype html>
-
-<html lang="en">
+<!--
```
Comment (solipet) [inline app/views/layouts/application.html.erb]: Again, remove comments

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -1,131 +1,208 @@
...
-    <link rel="stylesheet" href="/assets/uswds/css/uswds.min.css">
+    <title>login.gov Dashboard</title>
+    
+<!-- 
+Karla Rodriguez 
+removed link to uswds.min.css, added shortcut icons and added link to style guide in node modules 
+-->
+    <!-- 128x128 -->
+    <link rel="shortcut icon" type="image/ico" href="/assets/img/favicons/favicon.ico">
+    <link rel="icon" type="image/png" href="/assets/img/favicons/favicon.png">
```
Comment (solipet) [inline app/views/layouts/application.html.erb]: I'm looking at icons throughout and getting them to work in production with compile assets.

File: app/views/service_providers/all.html.erb [hunk 3]
```diff
@@ -1,6 +1,14 @@
-<h1>Apps</h1>
+<!--
```
Comment (solipet) [inline app/views/service_providers/all.html.erb]: Again, remove comments.

File: app/views/service_providers/edit.html.erb [hunk 4]
```diff
@@ -1,11 +1,16 @@
-<h1>Editing "<%= service_provider.friendly_name %>"</h1>
+<!--
```
Comment (solipet) [inline app/views/service_providers/edit.html.erb]: Again, remove comments

File: app/views/service_providers/index.html.erb [hunk 5]
```diff
@@ -1,6 +1,12 @@
-<h1>My apps</h1>
+<!--
```
Comment (solipet) [inline app/views/service_providers/index.html.erb]: Again, remove comments.

File: app/views/service_providers/new.html.erb [hunk 6]
```diff
@@ -1,7 +1,17 @@
-<h1>New test app</h1>
+<!--
```
Comment (solipet) [inline app/views/service_providers/new.html.erb]: Again, remove comments.

File: app/views/users/edit.html.erb [hunk 7]
```diff
@@ -1,7 +1,15 @@
-<h1>Edit user "<%= @user.email %>"</h1>
+<!--
```
Comment (solipet) [inline app/views/users/edit.html.erb]: Again, remove comments.

File: app/views/users/index.html.erb [hunk 8]
```diff
@@ -1,6 +1,15 @@
-<h1>Users</h1>
+<!--
```
Comment (solipet) [inline app/views/users/index.html.erb]: Again, remove comments.

File: app/assets/stylesheets/application.css.scss [hunk 9]
```diff
@@ -1,264 +1,4 @@
...
-.highlight .s1 { color: #198844 } /* Literal.String.Single */
-.highlight .ss { color: #F96A38 } /* Literal.String.Symbol */
-.highlight .bp { color: #CC342B } /* Name.Builtin.Pseudo */
-.highlight .fm { color: #3971ED } /* Name.Function.Magic */
-.highlight .vc { color: #3971ED } /* Name.Variable.Class */
-.highlight .vg { color: #3971ED } /* Name.Variable.Global */
-.highlight .vi { color: #CC342B } /* Name.Variable.Instance */
-.highlight .vm { color: #3971ED } /* Name.Variable.Magic */
-.highlight .il { color: #F96A38 } /* Literal.Number.Integer.Long */
+@import 'identity-style-guide/dist/assets/scss/styles';
```
[the remark this answers is not in the document]
Comment (karlaturcios) [reply app/assets/stylesheets/application.css.scss]: image

File: app/views/emails/index.html.erb [hunk 10]
```diff
@@ -1,4 +1,8 @@
-<h1>Emails</h1>
+<!--
```
[the remark this answers is not in the document]
Comment (karlaturcios) [reply app/views/emails/index.html.erb]: 👍🏼

File: app/views/groups/edit.html.erb [hunk 11]
```diff
@@ -1,7 +1,18 @@
-<h1>Edit group</h1>
+<!--
```
[the remark this answers is not in the document]
Comment (karlaturcios) [reply app/views/groups/edit.html.erb]: 👍🏼

File: app/views/groups/index.html.erb [hunk 12]
```diff
@@ -1,6 +1,14 @@
-<h1>Groups</h1>
+<!--
```
[the remark this answers is not in the document]
Comment (karlaturcios) [reply app/views/groups/index.html.erb]: 👍🏼

File: app/views/home/index.html.erb [hunk 13]
```diff
@@ -1,9 +1,15 @@
+<!--
```
[the remark this answers is not in the document]
Comment (karlaturcios) [reply app/views/home/index.html.erb]: 👍🏼

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (karlaturcios) [reply app/views/layouts/application.html.erb]: 👍🏼

File: app/views/layouts/application.html.erb (hunk 2 above)
Comment (karlaturcios) [reply app/views/layouts/application.html.erb]: 👍🏼

File: app/views/service_providers/all.html.erb (hunk 3 above)
Comment (karlaturcios) [reply app/views/service_providers/all.html.erb]: 👍🏼

File: app/views/service_providers/edit.html.erb (hunk 4 above)
Comment (karlaturcios) [reply app/views/service_providers/edit.html.erb]: 👍🏼

File: app/views/service_providers/index.html.erb (hunk 5 above)
Comment (karlaturcios) [reply app/views/service_providers/index.html.erb]: 👍🏼

File: app/views/service_providers/new.html.erb (hunk 6 above)
Comment (karlaturcios) [reply app/views/service_providers/new.html.erb]: 👍🏼

File: app/views/users/edit.html.erb (hunk 7 above)
Comment (karlaturcios) [reply app/views/users/edit.html.erb]: 👍🏼

File: app/views/users/index.html.erb (hunk 8 above)
Comment (karlaturcios) [reply app/views/users/index.html.erb]: 👍🏼

File: .idea/identity-dashboard.iml [hunk 14]
```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
[the remark this answers is not in the document]
Comment (karlaturcios) [reply .idea/identity-dashboard.iml]: done!

File: .codeclimate.yml [hunk 15]
```diff
@@ -63,6 +63,7 @@ plugins:
     enabled: true
     exclude_patterns:
     - 'spec/**/*'
+    - 'db/**/*'
```
Comment (achapm) [inline .codeclimate.yml]: We should stay in sync with the config here:
https://github.com/18F/identity-idp/blob/master/.codeclimate.yml
It excludes db/migrate/* and db/schema.rb which should address the same issues. This is what I have been instructed to do in the past.

Comment (karlaturcios) [reply .codeclimate.yml]: @solipet @blacktm FYI

Comment (karlaturcios) [reply .codeclimate.yml]: @achapm On the that [config](https://github.com/18F/identity-idp/blob/master/.codeclimate.yml) I see these additional items to exclude:
-  'node_modules/**/*'
- 'db/schema.rb'
- 'app/forms/password_form.rb'
- 'lib/user_flow_exporter.rb'
- 'lib/rspec/formatters/user_flow_formatter.rb'

I'll update those in this PR. What about db/migrate/* ? It is not listed there.

**Pull request merged**